packages/cozy-harvest-lib/test/helpers/accounts.spec.js

```diff
@@ -1,4 +1,4 @@
-import { mergeAuth, prepareAccountData } from 'helpers/accounts'
+import { getLabel, mergeAuth, prepareAccountData } from 'helpers/accounts'
```

Comment:
It is unfortunate that it is not possible to do 

`import { getLabel, mergeAuth, prepareData } as accounts from 'helpers/accounts'`

It means we have two things that are not compatible

* Keeping the namespace (good for clarity, shorter names in implementation file)
* Importing only what we need (presumably good for tree shaking)

Reply:
I will namespace this, that the best solution I think.